packages/api/test/metaplex-upload-v1.spec.js

```diff
@@ -0,0 +1,142 @@
+import assert from 'assert'
+import {
+  createTestUserWithFixedToken,
+  DBTestClient,
+  rawClient,
+} from './scripts/helpers.js'
+import { fixtures } from './scripts/fixtures.js'
+import { createCar } from './scripts/car.js'
+
+describe(' V1 - Metaplex Upload ', () => {
```

Comment:
```suggestion
describe('Metaplex Upload ', () => {
```

Also please remove "v1" from the filename. The v1 stuff will all be removed at some point.